index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    rel="stylesheet">
+</head>
+<body>
+    <div class="main-container">
+        <nav class="nav-bars">
+            <nav class="logo">
+              <h2>DNG</h2>
+            </nav>
+            <nav class="menu">
+              <nav><img src="photo/navs/Union.png" alt=""></nav>
+            </nav>
+        </nav>
+   
+        <section class="About-me">
+            <div class="greeting">
```

Comment:
- [ ] Please make sure that you have the same margins, padding, and width as the design fix the overflow x and y and make sure that your website is responsive

| Your design | Figma's |
| ---------------- | ----------- |
| ![image](https://user-images.githubusercontent.com/74744787/185757608-a412cb42-14eb-4f17-8dd8-93153f11537d.png) | ![image](https://user-images.githubusercontent.com/74744787/185757639-2f4f2398-ad87-4076-a99c-4976d720ca3e.png) |